src/gt4py/next/iterator/embedded.py

```diff
@@ -676,11 +676,23 @@ def _is_concrete_position(pos: Position) -> TypeGuard[ConcretePosition]:
 
 def _get_axes(
     field_or_tuple: LocatedField | tuple,
+    *,
+    ignore_zero_dims=False,
```

Comment:
From the PR it's not obvious to me how this is used. Please add an itir test.

Reply:
Are the changes in this file related to the newly added tests?

Reply:
Yes, they are required for "mixed" tuple args, e.g. an argument with type `(scalar, (field, scalar))`. This is the code path in `make_in_iterator` where this function is called.